kvmagent/kvmagent/plugins/vm_plugin.py

```
@@ -2412,11 +2412,42 @@ def __init__(self, task_spec, vm_uuid, volume, disk_name):
+
+    def _finish_detached_cache(self):
+        if self.progress == 100:
+            return True
+
+        cache = self.volume.cache
+        if not cache.installPath:
+            return False
+
+        try:
+            caches = qmp.execute_qmp_command(self.vm_uuid, "query-block-cache", raise_exception=False) or []
+        except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)